fe/fe-core/src/main/java/com/starrocks/server/NodeMgr.java

```diff
@@ -1105,6 +1113,28 @@ public long saveBrokers(DataOutputStream dos, long checksum) throws IOException
         return checksum;
     }
 
+    public long loadGlobalFunctions(DataInputStream dis, long checksum) throws IOException {
```

Comment:
Why add this function in NodeMgr again?

Reply:
Sorry, it must comes when merge. Those functions are useless now, I'll remove them.